packages/mds-ingest-service/@types/index.ts

```diff
@@ -122,16 +123,67 @@ export interface EventDomainModel extends RecordedColumn {
 
 export type EventDomainCreateModel = DomainModelCreate<Omit<EventDomainModel, keyof RecordedColumn>>
 
+/**
+ * Labels which can be used to annotate events.
+ */
+export interface DeviceLabel {
+  vehicle_id: string
+  vehicle_type: VEHICLE_TYPE
+  propulsion_types: PROPULSION_TYPE[]
+}
+
+export interface GeographyLabel {
+  geography_type: Nullable<string>
+  geography_id: UUID
+}
+
+export interface GeographiesLabel {
+  geographies: Array<GeographyLabel>
+}
+
+interface FlatGeographiesLabel {
+  geography_ids: UUID[]
+  geography_types: (string | null)[]
+}
+
+export interface LatencyLabel {
+  latency_ms: Timestamp
+}
+
+export interface TelemetryLabel {
+  telemetry_timestamp: Timestamp
+  telemetry_gps_lat: number
+  telemetry_gps_lng: number
+  telemetry_gps_altitude: Nullable<number>
+  telemetry_gps_heading: Nullable<number>
+  telemetry_gps_speed: Nullable<number>
+  telemetry_gps_accuracy: Nullable<number>
+  telemetry_charge: Nullable<number>
+}
```

Comment:
It seems like these labels exist elsewhere. Is this a copy or consolidation?

Reply:
Yeah, these are copied from the mds-stream-processor labelers. I think it makes sense for them to be defined here, and once we get a published `mds-ingest-service` I was going to delete the label typedefs from the stream processors and have them import these ones instead.

Reply:
https://github.com/lacuna-tech/lacuna-core/pull/647